### DISCUSSION TITLE
Distance calculation based on coordinates

### DISCUSSION THREAD

**Discussion by AndiLeni** [krisk/Fuse#724] [Q&A] [answered]
Hello,

is it possible to use latitude and longitude coordinates to sort a list by nearest distance?

**Comment by krisk** [comment, marked as answer]
Late reply — sorry about that.

No, Fuse.js is a text fuzzy-search library, so geographic distance isn't something it handles. For sorting by coordinates you'd want something like the [Haversine formula](https://en.wikipedia.org/wiki/Haversine_formula) or a library built for geospatial queries.
